wvlet-cli/src/main/scala/wvlet/lang/cli/WvletCompiler.scala

```diff
@@ -114,30 +117,59 @@ class WvletCompiler(
 
   end compiler
 
+  private def isSQLQuery(query: String): Boolean = {
+    val trimmed = query.trim.toLowerCase
+    trimmed.startsWith("select") || 
+    trimmed.startsWith("insert") || 
+    trimmed.startsWith("update") || 
+    trimmed.startsWith("delete") || 
+    trimmed.startsWith("create") || 
+    trimmed.startsWith("drop") || 
+    trimmed.startsWith("alter") || 
+    trimmed.startsWith("with")
```

Comment:
This detection logic is incomplete. Now that we have separate commands, compiel (for wvlet) or to_wvlet (for SQL), we can tell the input is SQL or Wvlet without looking the query string content. Remove this code and fix the inputUnit logic instead.